DronePhysicalStatus[] drones
CarPhysicalStatus[] cars
BillStatus[] bills
float64 score
EventMsg[] events